test/object-store/audit.cpp

```diff
@@ -52,13 +52,6 @@ using namespace std::string_literals;
 using Catch::Matchers::StartsWith;
 using nlohmann::json;
 
-static auto audit_logger =
```

Comment:
why did this need to change?

Reply:
I was trying to remove the instances of static loggers, but it doesn't matter here - reverted the change